functions/_tide_item_ruby.fish

```diff
@@ -0,0 +1,5 @@
+function _tide_item_ruby
+    if test -e .ruby-version
```

Comment:
As someone who would use this, a better choice would be to `test -e Gemfile || test -e Rakefile || test -e .ruby-version`. Not all of my Ruby projects have a `.ruby-version` file. Most of my projects have a Gemfile or Rakefile (you could also check for a `.gemspec`).

Reply:
I'd also like to advocate for testing for `Gemfile`, `Rakefile` or *.gemspec` along with `ruby --version` as it would support projects without a version specified as well as ASDF's `.tool-versions` file. Also, thanks for doing the work to add ruby support!

Reply:
I agree, I believe this line should be:

`if test -e Gemfile || test -e Rakefile || test -e *.gemspec || test -e .ruby-version || test -e .tool-versions`

Would love to see the code updated for for this feature to be merged!